uint32 slave
float32 value
